invenio_administration/errors.py

```diff
@@ -16,15 +16,27 @@ class InvalidResource(Exception):
 
     def __init__(self, resource, view):
         """Initialise error."""
-        super().__init__(_(f"Invalid resource {resource} configured for view {view}."))
+        super().__init__(
+            _(
+                "Invalid resource {resource} configured for view {view}.".format(
```

Comment:
why was this changed to use format function ?

Reply:
The process of automatically extracting string keys was failing for this file due to using string interpolation. Using format instead it works properly.